### ISSUE TITLE
How does this work with VuexPersist

### ISSUE THREAD

**Issue by karneaud** [localForage/localForage-compatibility-1-4#2]
Hello,

I'm experiencing a problem with iOS 10 for which you plug would be abele to rectify. But I'm not sure how you integrate it using vex-persist.so i pass the comparability promise? or the local forage instance still?

**Comment by tofumatt** [comment]
Sorry, I don't understand the issue. Could you please try to explain the issue you are experiencing using localForage? A code sample and/or error message would help us understand your issue better.

**Comment by tofumatt** [comment]
You might want to check with https://github.com/championswimmer/vuex-persist on the integration front. I don't know anything about that library but it does say it has localForage support.

**Comment by karneaud** [comment]
@tofumatt sorry for the erratic response. But it is a time sensitive issue and was scouring the internet trying to find and answer 

I'm using [this ](https://github.com/codecasts/spa-starter-kit) boilerplate which uses the localForage package to store information in the indexDB for auto login parameters/ scenarios on mobile phone.

The addition of the package `vuex-persist` happened because the storage delay issue and seemed  to resolved that matter.

The boilerplate works great on Android 5,7 and iPhone 11 but recently I encountered a situation in which iPhone 10.2.1 users had a problem with unexpected logouts right after they logged in. Only happened specifically to iPhone iOS 10 users. 

A Closer looked revealed that the key value entry for the indexDB was some how not  being kept when visiting another url or reloading the page. Research brought me to your package because I was inFact using localForage < 5 at the time. 

I figured that and upgrade would resolve the situation but so far hasn't which prompted me to look into your plugin. But I'm puzzled as to how it works and get the feeling `vuex-persist` is also something that does what your package does. 

I refer to this [stackoverflow](https://stackoverflow.com/questions/49585999/localforage-and-ios-10) refers to part of my problem. but the majority of work can be derived from the boilerplate mentioned
